src/components/DataFetcher/DataFetcher.js

```diff
@@ -4,17 +4,41 @@ import { get } from 'lodash';
 
 import jobPropTypes from '../Jobs/components/Job/jobPropTypes';
 import jobLogPropTypes from '../JobLogs/jobLogPropTypes';
+import { createUrl } from '../../utils';
 import {
   PROCESSING_IN_PROGRESS,
   PROCESSING_FINISHED,
   PARSING_IN_PROGRESS,
-  COMMITTED,
 } from '../Jobs/jobStatuses';
-import { DataFetcherContextProvider } from './DataFetcherContext';
+import { DataFetcherContextProvider } from '.';
```

Comment:
what is the benefit of such approach?
And it is not great that you import DataFetcherContextProvider from the file with different name.

Reply:
please add new line at the end